src/app/views/Welcome/Welcome.view.vue

```diff
@@ -5,9 +5,17 @@
     <welcome-gifs/>
 
     <v-container class="mt-10">
-      <span>Projeto criado para a materia de Engenharia de Software</span>
+      <span><i>Projeto criado para a matéria de Engenharia de Software.</i></span>
       <br/>
-      <span>O Giuliano vai por alguma coisa aqui porque eu não sei o que dizer!</span>
+      <br>
+      <span>
+      	Nosso Projeto Marvel consiste em utilizar a API Marvel Comics para ter acesso a informações sobre esse maravilhoso universo de super-heróis, <br/>
+      	assim sendo possivel encontrar todas as informações possíveis dos seus personagens favoritos.
+      </span>
+      <br></br>
+      <span>
+      	<b>Developed by:</b> Matheus Leite, Adriel Silva, Guilherme Mathias, Giuliano Montagnani
+      </span>
```

Comment:
You need to create a new *component* to this *description*. I recommend a *"WelcomeAppDescription"* name to this component.